cmd/istioctl/mixer.go

```diff
@@ -47,26 +47,73 @@ var (
 	mixerAPIServerAddr string
 
 	mixerCmd = &cobra.Command{
-		Use:          "mixer",
-		Short:        "Istio Mixer configuration",
+		Use:   "mixer",
+		Short: "Istio Mixer configuration",
+		Long: `
+The mixer configuration API allows users to configure all facets of a
+mixer deployment. The mixer configuration is organized in 'scope' ->
+'subject' -> 'rule' hierarchy. 'scope' and 'subject' are analogous to
+a manager > employee relationship. A manager ('scope') may configure
```

Comment:
maybe use parent child instead of manager analogy. there are too many managers around in the code :)